renovate.json

```diff
@@ -1,5 +1,6 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "updateInternalDeps":"true",
```

Comment:
Please revert all these package/renovate json changes

Reply:
> Can you test in a real repo? Not sure if we had any linked from when the capability was added previously

https://github.com/RahulGautamSingh/renovate/pull/2
Is working fine for Renovate.